components/collective-page/sections/Contribute.js

```diff
@@ -163,6 +164,8 @@ class SectionContribute extends React.PureComponent {
     const hasNoContributor = !this.hasContributors(contributors);
     const canContribute = collective.isActive && !isPastEvent(collective);
     const hasCustomContribution = !collective.settings?.disableCustomContributions;
+    const hasCryptoContribution =
+      collective.settings?.disableCryptoContributions === false && collective?.host.slug === 'opencollective';
```

Comment:
Don't hardcode, use a host setting.

By the way, if this was about activating on OSC, the slug would have been "opensource" not "opencollective".

Reply:
Thanks, makes sense now. I've added a host setting instead and checked for that instead. Let me know if everything looks okay now.